docs/activity.md

```diff
@@ -0,0 +1,241 @@
+# It's alive
+
+Let's bring our bird alive with code!
+
+<p align="center">
+<img src="/microbit-bird-activity/img/its-alive.gif" alt="It's alive gif">
+</p>
+
+
+## What is all this about?
+
+<img width="30%" alt="microbit-universal-hex logo" src="/microbit-bird-activity/img/bird-with-microbit.jpg" align="right">
+
+Your task is to think like a bird and programme the micro:bit to react to the world like a bird.
+
+The micro:bit can communicate wirelessly with other micro:bits using radio.
+There will be a secret micro:bit in the room broadcasting radio messages to
+all the birds indicating different events like "saying hello" o "there is a cat
+around".
+
+Apart from that, the micro:bit contains other sensors that your bird will react
+to, for example, birds don't like to be shaken!
```

Comment:
```suggestion
to, for example, birds don't like to be shaken! (we assume... no birds were harmed in the making of this workshop -though they may have been subjected to some unpleasant sounds from nearby micro:bits!)
```

Reply:
Love it! Added in 21ad6bda79cb278392ffcc3482dd4da27274a3cb